File: CS50-Psets/Week6/sentimental-mario-more/mario.py
# TODO
while True:
    # get user input
    try:
        height = int(input("Height: "))
        # limit height
        if height >= 1 and height <= 8:
            # for loop for row
            for i in range(height):
                # for loop for column
                for j in range(height):
                    if i + j < height - 1:
                        print(" ", end="")
                    else:
                        print("#", end="")
                print("  ", end="")

                # left alligned pyramid
                for k in range(i + 1):
                    print("#", end="")
                print("")
            break
    except ValueError:
        print("That's not an interger")

File: CS50-Psets/Week1/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    do
    {
        //prompts the user to enter the height
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);
    //for loop for column
    for (int i = 0; i < height; i++)
    {
        //for loop for row
        for (int j = 0; j < height; j++)
        {
            if (i + j < height - 1)
            {
                printf(" ");
            }
            else
            {
                printf("#");
            }
        }
        printf("\n");
    }
}